app/views/layouts/_searchbar.html.haml

```diff
@@ -1,5 +1,5 @@
-- if @lastaction == "show_list" && !session[:menu_click] && display_adv_search? && !@in_a_form
-  = render :partial => 'layouts/quick_search'
+- if @lastaction == 'show_list' && !session[:menu_click] && !@in_a_form
+  = render :partial => 'layouts/quick_search' if display_adv_search?
```

Comment:
This is confusing, quick search or adv search? Maybe it's just my brain's semantical engine but I need some explanation :pray:

Reply:
Yes, it may be confusing if you don't know what quick search is. When creating filters in Advanced Search, you always can check the checkbox "User will input the value". And later, when applying this filter, you will put your own value, in the modal window, then the filter is applied. And if Advanced Search is not supported, rendering anything related to quick search is unnecessary. Note that everything would work ok in the screen also without my `if display_adv_search?` in this line. However the original condition in line 1 is not ok if we want to display Search also without Advanced Search in some screens.

Reply:
@h-kataria can you help us with the quick search?

Reply:
@skateman What exactly is unclear for you regarding quick search? Or do you see any potential issue with it? If yes, let's improve the code. Otherwise this part of the code is clear to me and I don't need any help regarding quick search.